src/components/editMeasure/measureDetails/measureInformation/MeasureInformation.tsx

```diff
@@ -176,64 +182,93 @@ export default function MeasureInformation(props: MeasureInformationProps) {
     setToastOpen(open);
   };
 
+  const validateEndorser = (
```

Comment:
I thought we normally relied on the backend for this type of validation which then translated into the toasts

Reply:
@chubert-sb Yeah that's true, but we had a meeting yesterday with Nicole, Christina and Chris and a new validation from the frontend was added to the card.